src/bin/fullnode.rs

```diff
@@ -116,7 +110,27 @@ fn main() {
         }
         Some(port_number)
     } else {
-        None
+        match find_available_port_in_range((8899, 8999)) {
```

Comment:
magic numbers...

Reply:
Forgot to comment on this. I wanted suggestions. I can make it the full range for now if you think thats okay? There's a #define for the full range of ports.

Reply:
prolly want to start commenting on magics where we define them instead of where we use 'em

Reply:
You're right. I meant comment on here asking for feedback :)